### DISCUSSION TITLE
国内用这个方法，我用自己的域名以后，会不会被封号？

### DISCUSSION THREAD

**Discussion by bravohenry** [ChatGPTNextWeb/NextChat#824] [Q&A] [answered]
国内用这个方法，vercel搭建完，再用自己的域名以后，会不会被封号？

**Comment by Yidadaa** [comment, marked as answer]
不给别人用就不会。

**Reply by bravohenry** [reply]
谢谢您的答复！为什么给别人用会被封号呢？我人在国外，但是想搭建好给我的国内朋友们用。

**Reply by Yidadaa** [reply]
用的人多，你的域名就越容易被泄露，就会被防火墙污染

**Reply by Evolution-detector** [reply]
请问可以讲讲怎么算是域名泄露吗？我们尝试部署给团队用，就二十来人（估计大部分都是在同一个 IP 下用），绑了自有域名，用了三四天就收到网警的短信了。有没有什么措施可以去避免被墙呢？

**Reply by 14790897** [reply]
你是国内网站买的域名吗？

> 请问可以讲讲怎么算是域名泄露吗？我们尝试部署给团队用，就二十来人（估计大部分都是在同一个 IP 下用），绑了自有域名，用了三四天就收到网警的短信了。有没有什么措施可以去避免被墙呢？
你是国内网站买的域名吗？

**Reply by bravohenry** [reply]
> 请问可以讲讲怎么算是域名泄露吗？我们尝试部署给团队用，就二十来人（估计大部分都是在同一个 IP 下用），绑了自有域名，用了三四天就收到网警的短信了。有没有什么措施可以去避免被墙呢？

我在godaddy买的

**Reply by Evolution-detector** [reply]
不好意思，和朋友沟通后，更正之前表述的一些错误：收到的并不是网警的短信，是一个服务商的提醒，主要是告知我们网站访问速度异常了。
域名是在阿里云买的。

**Comment by yerochp** [comment]
为什么用上自己的域名后，就可以不用翻墙了，好神奇

**Reply by weykon** [reply]
对我也是，服务商的dns解析那里直接pass掉污染了哈哈哈哈哈？

**Comment by Xchat1** [comment]
请问我部署了该项目，已经设置了CODE，也重新部署了，请问访问密码显示为什么不生效呢，直接输入key 就可以授权使用，如何才能让访问密码code 生效呢
&nbsp;


1107590384
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年4月24日(星期一) 中午12:04
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; 
主题: Re: [Yidadaa/ChatGPT-Next-Web] 国内用这个方法，我用自己的域名以后，会不会被封号？ (Discussion #824)





 
不好意思，和朋友沟通后，更正之前表述的一些错误：收到的并不是网警的短信，是一个服务商的提醒，主要是告知我们网站访问速度异常了。
 域名是在阿里云买的。

**Comment by Xchat1** [comment]
请问我部署了该项目，已经设置了CODE，也重新部署了，请问访问密码显示为什么不生效呢，直接输入key 就可以授权使用，如何才能让访问密码code 生效呢


1107590384
***@***.***



&nbsp;

**Reply by bravohenry** [reply]
可以再描述清楚点么

**Reply by alexdemoon** [reply]
对我也是，在vercel重新部署了这个项目后，就不用输入密码就可以直接使用了，

**Reply by weykon** [reply]
去看下代码是不是填写了key就不检验code了

**Comment by Xchat1** [comment]
{
&nbsp; "error": {
&nbsp; &nbsp; "message": "Invalid URL (POST /api/chat-stream)",
&nbsp; &nbsp; "type": "invalid_request_error",
&nbsp; &nbsp; "param": null,
&nbsp; &nbsp; "code": null
&nbsp; }
}





1107590384
***@***.***



&nbsp;
